pkg/process/metadata/workloadmeta/grpc.go

```diff
@@ -50,9 +52,12 @@ func NewGRPCServer(config config.ConfigReader, extractor *WorkloadMetaExtractor)
 	l := &GRPCServer{
 		config:    config,
 		extractor: extractor,
-		server: grpc.NewServer(grpc.KeepaliveParams(keepalive.ServerParameters{
-			Time: 10 * time.Second,
-		})),
+		server: grpc.NewServer(
+			grpc.KeepaliveParams(keepalive.ServerParameters{
+				Time: 10 * time.Second,
```

Comment:
Optional nit: since we're putting the connection timeout in a const, maybe do the same for the keepalive as well?

Reply:
Great suggestion! Done